sdks/python/apache_beam/runners/worker/sdk_worker_main.py

```diff
@@ -78,8 +67,7 @@ def do_GET(self):  # pylint: disable=invalid-name
         self.send_header('Content-Type', 'text/plain')
         self.end_headers()
 
-        for line in StatusServer.get_thread_dump():
-          self.wfile.write(line.encode('utf-8'))
+        self.wfile.write(thread_dump())
```

Comment:
Lets add a jira to clean `StatusServer` from here completely once we have rolled out Debug capture.

Reply:
created https://issues.apache.org/jira/browse/BEAM-9165